solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceConsumerProperties.java

```diff
@@ -1,32 +1,92 @@
 package com.solace.spring.cloud.stream.binder.properties;
 
 import com.solacesystems.jcsmp.EndpointProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 
 import java.util.concurrent.TimeUnit;
 
+@ConfigurationProperties("spring.cloud.stream.solace.default.consumer")
```

Comment:
* This raises a compiler warning that the configuration isn't registered anywhere. Add a `@SuppressWarnings("ConfigurationProperties")` to the class.
* Make `SolaceExtendedBindingProperties.DEFAULTS_PREFIX` a private-protected constant and use it as the prefix for the `@ConfigurationProperties` value. i.e. `@ConfigurationProperties(DEFAULTS_PREFIX + ".consumer")`

Reply:
Done